Update scripting_player_input.rst (#8722)
godotengine/godot-docs: getting_started/step_by_step/scripting_player_input.rst

Finally, we use the ``direction`` as a multiplier when we update the node's
``rotation``: ``rotation += angular_speed * direction * delta``.

> Comment out the lines ``var velocity = Vector2.UP.rotated(rotation) * speed`` and ``position += velocity * delta`` with the code below:

Reviewer: ```suggestion
Comment out the lines ``var velocity = Vector2.UP.rotated(rotation) * speed`` and ``position += velocity * delta`` like this:
```
the phrasing "with the code below" sounds weird to me.

.. tabs::

 .. code-tab:: gdscript GDScript

    #var velocity = Vector2.UP.rotated(rotation) * speed
	
    #position += velocity * delta

 .. code-tab:: csharp C#

>     // var velocity = Vector2.Up.Rotated(Rotation) * _speed;
> 
>     // Position += velocity * (float)delta;

Reviewer 2: ```suggestion
    //var velocity = Vector2.Up.Rotated(Rotation) * _speed;

    //Position += velocity * (float)delta;
```
Commented out code has no leading space, like in GDScript

-------------------------

To only move when pressing a key, we need to modify the code that calculates the

> velocity. Uncomment the code and replace the line starting with ``var velocity`` with the code below.

Author: @skyace65 I've made a simple update that accounts for both comments.
